src/sections/TrustedCompaniesSection/TrustedCompaniesSection.tsx

```diff
@@ -346,6 +346,12 @@ const COMPANIES: Company[] = [
     src: gowtherNdoe,
     logoClass: 'max-h-9',
   },
+  {
```

Comment:
you need an .png file to import to have a src here

Reply:
Yes, I added it